errors/sharp-missing-in-production.md

```diff
@@ -2,13 +2,34 @@
 
 #### Why This Error Occurred
 
-The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment. For a production environment using `next start`, it is strongly recommended you install [`sharp`](https://www.npmjs.com/package/sharp) by running `yarn add sharp` in your project directory.
+The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment.
+
+- For a production environment using `next start`, it is strongly recommended you install [`sharp`](https://www.npmjs.com/package/sharp).
 
 You are seeing this error because Image Optimization in production mode (`next start`) was detected.
 
+- For a production environment using `output: "standalone"`, you must install [`sharp`](https://www.npmjs.com/package/sharp).
+
+You are seeing this error because Image Optimization in standalone mode (`output: "standalone"`) was detected.
+
 #### Possible Ways to Fix It
 
-- Install `sharp` by running `yarn add sharp` in your project directory and then reboot the server by running `next start` again
+- Install `sharp` by running one of the following commands in your project directory:
+
+```bash
+npm i sharp
+```
+
+```bash
+yarn add sharp
+```
+
+```bash
+pnpm add sharp
+```
+
+Then, reboot the server by running `next start` again, or if you have `output: "standalone"` enabled, rebuild your project with `next build` and reboot the server with `node server.js`.
```

Comment:
I actually think we should recommend `next build` either way since we generate blur placeholders at build time.

```suggestion
Then, build your project with `next build`. Finally, restart the server with either `next start` for production mode or `node server.js` for standalone mode.
```